scripts/local-install.sh

```diff
@@ -53,14 +53,16 @@ COMPRESSED_PACKAGE="$(tail -1 /tmp/pack.out)"
 tar -xzf "${COMPRESSED_PACKAGE}"
 rm "${COMPRESSED_PACKAGE}"
 mv package "${PACKAGE_DIR}"
-PACKAGE_NAME=$(cat ./package.json | jq -r '.name')
+PACKAGE_NAME=$(jq -r '.name' < ./package.json)
```

Comment:
This syntax is equivalent, but it was an unnecessary use of `cat` and causing a shellcheck warning.